axe_selenium_python/axe.py

```diff
@@ -100,7 +100,9 @@ def write_results(self, data, name="results.json"):
         :param name: Name of file to be written to.
         :param output: JSON object.
         """
-        with open(name, "w", encoding="utf8") as f:
+        filepath = os.path.join(os.getcwd(), name)
```

Comment:
This hard-codes writing the file to the current working directory. Could there be a reason to want to write the file elsewhere? If so, it might make sense for `write_results` to take a path. Then the user can decide if they want it to be relative, or absolute, and can decide the destination. You could still default to current working directory if only a file name is provided, but you'd also need to handle when the destination does not exist. You can see an example of this in pytest-html https://github.com/pytest-dev/pytest-html/blob/master/pytest_html/plugin.py#L89-L90

Reply:
Thanks @davehunt 

Opened a new issue to address this.